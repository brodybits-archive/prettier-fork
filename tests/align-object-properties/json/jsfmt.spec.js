run_spec(__dirname, ["json", "json-stringify"], {
  alignObjectProperties: true
});
run_spec(__dirname, ["json5"], {
  alignObjectProperties: true
});
