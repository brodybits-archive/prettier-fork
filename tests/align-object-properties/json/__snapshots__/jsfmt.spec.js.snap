// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`o1.json 1`] = `
====================================options=====================================
alignObjectProperties: true
parsers: ["json", "json-stringify"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
{
  2:3,
  "k-2":2,
  keyasdf:3
}

=====================================output=====================================
{
  2: 3,
  "k-2": 2,
  "keyasdf": 3
}

================================================================================
`;

exports[`o1.json 2`] = `
====================================options=====================================
alignObjectProperties: true
parsers: ["json5"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
{
  2:3,
  "k-2":2,
  keyasdf:3
}

=====================================output=====================================
{
  2: 3,
  "k-2": 2,
  keyasdf: 3
}

================================================================================
`;
